### ISSUE TITLE
Design Page Entity in D8

### ISSUE THREAD

**Issue by dannylamb** [Islandora-CLAW/CLAW#340]
Depends on #334 and #335 and #339.

Do not attempt this until those have been resolved. Outcome is a plan for implementation. Like the other tickets, think in terms of classes and base fields. What would the annotation look like?  What relationships are needed for it to be part of a book?

**Comment by dannylamb** [comment]
Closing since sprint is over.  We can open another ticket to 'revisit' this concept later if required.
